src/pages/BorrowContent.jsx

```diff
@@ -0,0 +1,580 @@
+          1000000,
+      ) / 100;
+    setMaxGARD(max);
+    if (mGARD > max) {
+      setGARD(max < 1 ? 1 : max);
+    }
+  };
+
+  const handleMaxCollateral = () => {
+      setCollateral(maxCollateral)
+      let max =
+        Math.trunc(
+          (100 *
+            ((algosToMAlgos(price) * algosToMAlgos(maxCollateral)) /
+              1000000)) /
+            1.4 /
+            1000000,
+        ) / 100;
+      setMaxGARD(max);
+      if (mGARD > max) {
+        setGARD(max < 1 ? 1 : max);
+      }
+      console.log("collateral" ,cAlgos)
+    }
+  
+
+
+  const handleBorrowChange = (event) => {
+    setGARD(
+      event.target.value === ""
+        ? ""
+        : Number(event.target.value) < 1
+        ? 1
+        : Number(event.target.value),
+    );
+    let max = mAlgosToAlgos(
+      getWalletInfo()["amount"] -
+        calcDevFees(algosToMAlgos(mGARD)) -
+        307000 -
+        100000 * (getWalletInfo()["assets"].length + 4),
+    ).toFixed(3);
+    setMaxCollateral(max);
+    if (isNaN(cAlgos)) {
+      console.log("heyy");
+      return;
+    }
+    if (cAlgos > max) {
+      setCollateral(max);
+    }
+  };
+
+  const handleMaxBorrow = () => {
+    setGARD(maxGARD)
+    let max = mAlgosToAlgos(
+      getWalletInfo()["amount"] -
+        calcDevFees(algosToMAlgos(maxGARD)) -
+        307000 -
+        100000 * (getWalletInfo()["assets"].length + 4),
+    ).toFixed(3);
+    setMaxCollateral(max);
+    if (isNaN(cAlgos)) {
+      console.log("heyy");
+      return;
+    }
+    if (cAlgos > max) {
+      setCollateral(max);
+    }
+    console.log("gard" ,mGARD)
+  }
+
+  var sessionStorageSetHandler = function (e) {
+    setLoadingText(JSON.parse(e.value));
+  };
+  document.addEventListener("itemInserted", sessionStorageSetHandler, false);
+  var details = [
+    {
+        title: "Collateral",
+        val: `${cAlgos === "" ? "...": `$${(cAlgos * supplyPrice).toFixed(2)}`}`,
+        hasToolTip: true,
+    },
+    {
+        title: "Received DAI",
+        val: `${0.00}%`,
+        hasToolTip: true,
+    },
+    {
+        title: "Liquidation Price",
+        val: `${getMinted() == null || getCollateral() == null
+          ? "..."
+          : displayLiquidationPrice()}`,
+        hasToolTip: true,
+    },
+    {
+        title: "Bitcoin Factor",
+        val: `${0.00}%`,
+        hasToolTip: true,
+    },
+    {
+        title: "ETH exposure",
+        val: `${0.00}%`,
+        hasToolTip: true,
+    },
+    {
+        title: "Stability Fee",
+        val: `${getMinted() == null || getCollateral() == null
+          ? "..."
+          : displayFees()}`,
+        hasToolTip: true,
+    },
+    {
+        title: "Liquidation ratio",
+        val: `${getMinted() == null || getCollateral() == null
+          ? "..."
+          : displayRatio()}`,
+        hasToolTip: true,
+    },
+    {
+        title: "DAI avaible from ETH",
+        val: `${0.00}%`,
+        hasToolTip: true,
+    },
+]
+
+var supplyDetails = [
```

Comment:
Same comments here as for the other supplyDetails and borrowDetails. I.E. Why is borrow limit under supplyDetails and what the heck are borrow rewards and supply rewards?

Reply:
I agree, I don't really know why the borrow limit is under supply details, but I made it this way because it was part of the design. I added a comment on figma a couple of days ago about this, but I can change it now because I think it makes much more sense with borrow limit and supply limit in their respective details.